src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.test.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, cleanup, waitForDomChange } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import ReportsTrafficSources from './ReportsTrafficSources';
+import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../../services/pure-di';
+
+const emptyResponse = { success: false, error: new Error('Dummy error') };
+const fullResponse = {
+  success: true,
+  value: [
+    {
+      sourceName: 'Email',
+      quantity: 2000,
+    },
+    {
+      sourceName: 'Social',
+      quantity: 1000,
+    },
+  ],
+};
+
+describe('reports traffic sources', () => {
+  afterEach(cleanup);
+  it('should datahub fail', async () => {
```

Comment:
It is not clear the meaning of the test. It should be readable as a sentence, in this case: `reports traffic sources should datahub fail`, what do you think about: `ReportTrafficSources component should deal with DataHub failure`?

Reply:
i use in describe reports traffic source already, its not better put `should deal with Datahub failure` ?

Reply:
No entendí, solo creo que la oración `it should datahub fail` tiene sentido y que debería ponerse algo que lo tenga, solo puse un ejemplo, otra frase también me parece bien.